pkg/dbg/profile_dump.go

```diff
@@ -28,11 +28,11 @@ func DumpGoMemoryTrace() {
 func DumpGoProfile() error {
 	trace := make([]byte, 5120*1024)
 	len := runtime.Stack(trace, true)
-	return ioutil.WriteFile(path+time.Now().String()+".stack", trace[:len], 0644)
+	return ioutil.WriteFile(path+time.Now().Format("2006-01-02T15:04:05.999999-0700MST")+".stack", trace[:len], 0644)
```

Comment:
Use constants instead of hardcoding formats. Also time.RFC3339 is a similar format.